docs/book/content/theory/demographics.md

```diff
@@ -121,26 +121,28 @@ kernelspec:
   $$
   \boldsymbol{\Omega}^2 =
     \begin{bmatrix}
-      * & *  & * & \cdots & * & * & *\\
-      * & * & * & \cdots & * & * & * \\
-      0 & * & * & \cdots & 0 & 0 & 0 \\
-      \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots \\
-      0 & 0 & 0 & \cdots & *  & * & 0 \\
-      0 & 0 & 0 & \cdots & 0 & * & *
-    \end{bmatrix}; ~~~
-    \boldsymbol{\Omega}^{S+E-1} =
-    \begin{bmatrix}
-      * & *  & * & \cdots & * & * & *\\
-      * & * & * & \cdots & * & * & * \\
-      * & * & * & \cdots & * & * & * \\
+      * & *  & * & * & \cdots & * & * & * & *\\
+      * & * & * & * & \cdots & * & * & * & * \\
+      * & * & * & 0 & \cdots & 0 & 0 & 0 & 0 \\
       \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots \\
-      * & * & * & \cdots & *  & * & * \\
-      0 & 0 & 0 & \cdots & 0 & * & *
-    \end{bmatrix}
-    $$
+      0 & 0 & 0 & 0 & \cdots & * & * & * & 0 \\
+      0 & 0 & 0 & 0 & \cdots & 0 & * & * & *
+    \end{bmatrix} $$
+    
+  $$
+  \boldsymbol{\Omega}^{S+E-2} =
+  \begin{bmatrix}
+    * & *  & * & \cdots & * & * & *\\
+    * & * & * & \cdots & * & * & * \\
+    * & * & * & \cdots & * & * & * \\
+    \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots \\
+    * & * & * & \cdots & * & * & 0 \\
+    0 & * & * & \cdots & * & * & *
+  \end{bmatrix}
+  $$
 
   $$
-  \boldsymbol{\Omega}^{S+E} =
+  \boldsymbol{\Omega}^{S+E-1} =
```

Comment:
@prrathi Can you explain what you've done with these matrices?  Why no Omega^E+S?

Reply:
@jdebacker For the change with Omega^2, the first entry of third row should be nonzero (because second element of third row and second element of first column are both nonzero), and every row there on out has three nonzero elements instead of two. I thus included more values in the matrix to depict this clearly. Following this pattern, the Omega^S+E-1 instead of Omega^S+E is the first with all nonzero values. I manually tested with smaller matrices to make sure this was right.

Reply:
@prrathi This sounds good to me.  Let me ping @rickecon to be sure since he's worked out this math more times than anyone.  @rickecon Can you confirm what @prrathi's done here?

Reply:
This is correct. I clarified this a little more in some changes I submitted to @prrathi's branch.